lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -260,7 +260,7 @@ function init(newConfig, callback) {
 }
 
 function clear(callback) {
-    dbService.dropDatabase(callback);
+    dbService.db.db.dropDatabase(callback);
```

Comment:
Yes, it is, due to a horrible choice of attribute names (by me). The first db is the attribute of the module containing the connection; the second one is the `db` attribute of the `connection` object, that holds a pointer to the actual database (as can be used in Mongo's command line interpreter).